### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by carreraprogrammer**
This branch was created in order to add some new tables and add foreign key columns to the existing animal's table to model one-to-many relationships and afterward, use JOIN to query the data:

- [x] Create a table named owners with the following columns:
- id: integer (set it as autoincremented PRIMARY KEY)
- full_name: string
- age: integer

- [x] Create a table named species with the following columns:
- id: integer (set it as autoincremented PRIMARY KEY)
- name: string

- [x] Modify animals table:
- Make sure that id is set as autoincremented PRIMARY KEY
- Remove column species
- Add column species_id which is a foreign key referencing species table
- Add column owner_id which is a foreign key referencing the owners table

- [x] Remember all this goes in the schema.sql file.

- [x] Insert the following data into the owners table:
- Sam Smith 34 years old.
- Jennifer Orwell 19 years old.
- Bob 45 years old.
- Melody Pond 77 years old.
- Dean Winchester 14 years old.
- Jodie Whittaker 38 years old.

- [x] Insert the following data into the species table:
- Pokemon
- Digimon

- [x] Modify your inserted animals so it includes the species_id value:
- If the name ends in "mon" it will be Digimon
- All other animals are Pokemon

- [x] Modify your inserted animals to include owner information (owner_id):
- Sam Smith owns Agumon.
- Jennifer Orwell owns Gabumon and Pikachu.
- Bob owns Devimon and Plantmon.
- Melody Pond owns Charmander, Squirtle, and Blossom.
- Dean Winchester owns Angemon and Boarmon

- [x] Remember these insertions and modifications happen in the data.sql file.

- **Write queries (using JOIN) to answer the following questions:**

- [x] What animals belong to Melody Pond?

query_19

- [x] List of all animals that are pokemon (their type is Pokemon).

query_20

- [x] List all owners and their animals, remember to include those that don't own any animal

query_21

- [x] How many animals are there per species?

query_22

- [x] List all Digimon owned by Jennifer Orwell.

query_23

- [x] List all animals owned by Dean Winchester that haven't tried to escape.

Answer: There aren't animals of Dean Winchester that haven't tried to escape

query_24a
query_24b

- [x] Who owns the most animals?

query_25

- [x] Remember all these should be written in queries.sql file.
- [x] **Take a screenshot of the results of your queries.**
- [x] Remember to add the screenshot with the results of your queries to your Pull Request description.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -17,3 +17,60 @@ ALTER TABLE animals
...
+BEGIN;
+
+ALTER TABLE animals
+ADD COLUMN owner_id INT;
+
+ALTER TABLE animals
+ADD CONSTRAINT dk_owner_id
+FOREIGN KEY(owner_id)
+REFERENCES owners(id);
+
```
Comment (CollinsTatang) [inline schema.sql:75]: ### **Optional changes**
I agree with you on the code above 
but you should write it, with just three lines of code 
### **Example**
```
ALTER TABLE REMOVE species;
ALTER TABLE animals ADD species_id int REFERENCES species(id);
ALTER TABLE animals ADD owner_id int REFERENCES owners(id);

 ```

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (carreraprogrammer) [reply schema.sql:75]: Hello @CollinsTatang, Thanks for the review! I will add all the changes that you recommend and continue to merge 😁

**Review by rotimiazeez — COMMENTED** [review]
# Hi @carreraprogrammer 👋🏼, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project is already merged. Please ensure the PR you are submitting is open 🙏🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request merged**